converters/fromZigbee.js

```diff
@@ -4272,6 +4272,90 @@ const converters = {
             };
         },
     },
+    ZG2819S_change_color: {
+        cluster: 'lightingColorCtrl',
+        type: 'commandMoveToColor',
+        convert: (model, msg, publish, options, meta) => {
+            return {
+                [getProperty('color', msg, model)]: {
+                    x: precisionRound(msg.data.colorx / 65535, 3),
+                    y: precisionRound(msg.data.colory / 65535, 3),
+                },
+                [getProperty('click', msg, model)]: 'color_wheel',
+                [getProperty('transition_time', msg, model)]: msg.data.transtime,
+            };
+        },
+    },
+    ZG2819S_change_brightness: {
+        cluster: 'genLevelCtrl',
+        type: 'commandStepWithOnOff',
+        convert: (model, msg, publish, options, meta) => {
+            return {
+                [getProperty('click', msg, model)]: (msg.data.stepmode == 0) ? 'brightness_up' : 'brightness_down',
+                [getProperty('step_mode', msg, model)]: msg.data.stepmode,
+                [getProperty('step_size', msg, model)]: msg.data.stepsize,
+                [getProperty('transition_time', msg, model)]: msg.data.transtime,
+            };
+        },
+    },
+    ZG2819S_command_on: {
+        cluster: 'genOnOff',
+        type: 'commandOn',
+        convert: (model, msg, publish, options, meta) => {
+            // The device sends this command for all four group IDs.
+            // Only forward for the first group.
+            if (msg.groupID !== 46337) {
+                return null;
+            }
+            return {[getProperty('click', msg, model)]: 'on'};
```

Comment:
The current convention is to use `action` instead of `click` so please change this to `return {action: 'on'}`